ep/build-velox/src/build_velox.sh

```diff
@@ -96,23 +96,26 @@ function compile {
   fi
 
   COMPILE_OPTION="-DVELOX_ENABLE_PARQUET=ON -DVELOX_BUILD_TESTING=OFF -DVELOX_BUILD_TEST_UTILS=OFF -DVELOX_ENABLE_DUCKDB=OFF -DVELOX_ENABLE_PARSE=OFF"
-  if [ $ENABLE_BENCHMARK == "ON" ]; then
-    COMPILE_OPTION="$COMPILE_OPTION -DVELOX_BUILD_BENCHMARKS=ON"
-  fi
-  if [ $ENABLE_TESTS == "ON" ]; then
-    COMPILE_OPTION="$COMPILE_OPTION -DVELOX_BUILD_TESTING=ON "
-  fi
   if [ $ENABLE_HDFS == "ON" ]; then
     COMPILE_OPTION="$COMPILE_OPTION -DVELOX_ENABLE_HDFS=ON"
   fi
-  if [ $ENABLE_ABFS == "ON" ]; then
-    COMPILE_OPTION="$COMPILE_OPTION -DVELOX_ENABLE_ABFS=ON"
-  fi
   if [ $ENABLE_S3 == "ON" ]; then
     COMPILE_OPTION="$COMPILE_OPTION -DVELOX_ENABLE_S3=ON"
   fi
-  if [ $ENABLE_GCS == "ON" ]; then
-    COMPILE_OPTION="$COMPILE_OPTION -DVELOX_ENABLE_GCS=ON"
+  # If ENABLE_BENCHMARK == ON, Velox disables tests and connectors
+  if [ $ENABLE_BENCHMARK == "OFF" ]; then
+    if [ $ENABLE_TESTS == "ON" ]; then
```

Comment:
This seems to be a regression of https://github.com/oap-project/gluten/pull/4176. Script `./dev/package.sh` is broken. cc @zhouyuan @surnaik

Reply:
This is due to a recent commit in Velox - 

https://github.com/facebookincubator/velox/commit/f2cdf813be50cc165a104b93cabbb56319bd4822